common/const/strengths.ts

```diff
@@ -1,8 +1,22 @@
 import {TypeT} from '../../common/types/cards'
 
-export const STRENGTHS: Record<TypeT, Array<TypeT>> = {
-	balanced: [],
+type strengths = {
+	readonly any: Array<TypeT>
+	readonly balanced: Array<TypeT>
+	readonly builder: Array<TypeT>
+	readonly explorer: Array<TypeT>
+	readonly farm: Array<TypeT>
+	readonly miner: Array<TypeT>
+	readonly prankster: Array<TypeT>
+	readonly pvp: Array<TypeT>
+	readonly redstone: Array<TypeT>
+	readonly speedrunner: Array<TypeT>
+	readonly terraform: Array<TypeT>
+}
```

Comment:
This is the evil option, but there is nothing I like more than a good old metatype 
```suggestion
type Strengths = {
	readonly [Type in TypeT]: Array<TypeT>
}
```